styles/styles.css

```diff
@@ -14,13 +14,15 @@
   /* colors */
   --link-color: #035fe6;
   --link-hover-color: #136ff6;
-  --background-color: #fff;
-  --overlay-background-color: #eee;
+  --background-color: #000;
+  --overlay-background-color: #000;
   --highlight-background-color: #ccc;
-  --text-color: #000;
+  --text-color: white;
 
   /* fonts */
-  --body-font-family: 'helvetica neue', helvetica, ubuntu, roboto, noto, sans-serif;
+
+  /* --body-font-family: 'helvetica neue', helvetica, ubuntu, roboto, noto, sans-serif; */
+  --body-font-family: 'Playfair display', 'Didot', 'Raleway', roboto, noto, sans-serif;
```

Comment:
If you can't import Playfair Display, can you replace it with another font that is free and looks close/similar?

Reply:
Imported and committed